client/src/components/Loading/Loading.tsx

```diff
@@ -1,5 +1,5 @@
 import styles from './Loading.module.scss';
-import logo from '../../assets/Temp_Roadwatch_logo.svg';
+import logo from '../../assets/Updated_Roadwatch_Logo.svg';
```

Comment:
I think this import should be `import logo from '../../assets/Updated_RoadWatch_Logo.svg';` with a capital 'W'. Github is yelling at you for it.